articles/libraries/auth0-swift/index.md

```diff
@@ -112,14 +140,16 @@ Auth0
    }
 ```
 
-#### Sign Up with database connection
+### Signing Up with database connection
```

Comment:
Avoid using words which imply a level of difficulty in a task (even if it is easy :)